#INPUT
string newString
---
#OUTPUT
string changedString
